params/config.go

```diff
@@ -615,7 +616,9 @@ type ChainConfig struct {
 	BrunoBlock      *big.Int `json:"brunoBlock,omitempty" toml:",omitempty"`      // brunoBlock switch block (nil = no fork, 0 = already activated)
 
 	// EIP-3675: Upgrade consensus to Proof-of-Stake
-	TerminalTotalDifficulty *big.Int `json:"terminalTotalDifficulty,omitempty"` // The merge happens when terminal total difficulty is reached
+	TerminalTotalDifficulty *big.Int        `json:"terminalTotalDifficulty,omitempty"` // The merge happens when terminal total difficulty is reached
```

Comment:
as you can see you using * which makes the field optional, be careful, see comment above about nil-checking